gpii/node_modules/ontologyHandler/test/node/OntologyHandlerTests.js

```diff
@@ -79,6 +79,159 @@ https://github.com/gpii/universal/LICENSE.txt
+            edges: [["A", "B"], ["B", "A"], ["B", "C"], ["C", "B"]],
+            tests: [
+                {
+                    desc: "3v2e A to B",
+                    start: "A",
+                    end: "B",
+                    path: ["A", "B"]
+                }, {
+                    desc: "3v2e A to C",
+                    start: "A",
+                    end: "C",
+                    path: ["A", "B", "C"]
+                }
+            ]
+        }, {
+            name: "Square",
+            edges: [["A", "B"], ["B", "A"], ["A", "C"], ["C", "A"],
+                    ["C", "D"], ["D", "C"], ["D", "B"], ["B", "D"]],
+            tests: [
+                {
+                    desc: "Square A to B",
+                    start: "A",
+                    end: "B",
+                    path: ["A", "B"]
+                }, {
+                    desc: "Square A to D",
+                    start: "A",
+                    end: "D",
+                    path: ["A", "B", "D"]
+                }, {
+                    desc: "Square A to C",
+                    start: "A",
+                    end: "C",
+                    path: ["A", "C"]
+                }
+            ]
+        }, {
+            name: "Tetrahedron",
+            edges: [["A", "B"], ["B", "A"], ["A", "C"], ["C", "A"],
+                    ["C", "D"], ["D", "C"], ["D", "B"], ["B", "D"],
+                    ["A", "D"], ["D", "A"], ["B", "C"], ["C", "B"]],
+            tests: [
+                {
+                    desc: "Tetrahedron A to B",
+                    start: "A",
+                    end: "B",
+                    path: ["A", "B"]
+                }, {
+                    desc: "Tetrahedron A to D",
+                    start: "A",
+                    end: "D",
+                    path: ["A", "D"]
+                }, {
+                    desc: "Tetrahedron A to C",
+                    start: "A",
+                    end: "C",
+                    path: ["A", "C"]
+                }
+            ]
+        }, {
+            name: "4 vertices, 2 disconnected edges",
+            edges: [["A", "B"], ["B", "A"], ["A", "C"], ["C", "A"],
+                    ["C", "D"], ["D", "C"], ["A", "D"], ["D", "A"]],
+            tests: [
+                {
+                    desc: "4vert2dis A to B",
+                    start: "A",
+                    end: "B",
+                    path: ["A", "B"]
+                }, {
+                    desc: "4vert2dis A to D",
+                    start: "A",
+                    end: "D",
+                    path: ["A", "D"]
+                }, {
+                    desc: "4vert2dis A to C",
+                    start: "A",
+                    end: "C",
+                    path: ["A", "C"]
+                }, {
+                    desc: "4vert2dis B to D",
+                    start: "B",
+                    end: "D",
+                    path: ["B", "A", "D"]
+                }
+            ]
```

Comment:
Add a further check here: 
Consider if edges is [A, B], [B, A], [C, D], [D, C] and you ask for a path from A to D